interview-questions.md

```diff
@@ -4,57 +4,123 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: Create, Read, Upload, Delete
 
-  Researched answer:
+  Researched answer: "In computer programming, create, read, update, and delete (CRUD) are the four basic functions of persistent storage."
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: put, patch, parse(?), get, delete
 
-  Researched answer:
+  Researched answer: "The primary or most-commonly-used HTTP verbs (or methods, as they are properly called) are POST, GET, PUT, PATCH, and DELETE. "
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
   Your answer:
 
+  class App extends Component {
+    render() {
+      return (
+          <p>Hello World</p>
+        )
+    }
+  }
+
+
   Researched answer:
 
+  same as above
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow